airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -416,6 +416,10 @@ class Plans(IncrementalStripeStream):
     def path(self, **kwargs):
         return "plans"
 
+    def request_params(self, stream_slice: Mapping[str, Any] = None, **kwargs):
+        params = super().request_params(stream_slice=stream_slice, **kwargs)
+        params["expand[]"] = ["data.tiers"]
```

Comment:
Could you please reflect this change in the stream schema as well? As far as I understand, it does not correspond to the docs. (at least 2 fields missing)